fury/gltf.py

```diff
@@ -398,3 +399,404 @@ def load_camera(self, camera_id, transform_mat):
+        count += 1
+    material = None if tcoords is None else 0
+    prim = get_prim(vertex, index, color, tcoord, normal, material)
+    return prim, boffset, count
+
+
+def write_scene(gltf, nodes):
+    """Create scene
+
+    Parameters
+    ----------
+    gltf : GLTF2
+        Pygltflib GLTF2 object
+    nodes : list
+        List of node indices.
+    """
+    scene = gltflib.Scene()
+    scene.nodes = nodes
+    gltf.scenes.append(scene)
+
+
+def write_node(gltf, mesh=None, camera=None):
+    """Create node
+
+    Parameters
+    ----------
+    gltf : GLTF2
+        Pygltflib GLTF2 object
+    mesh : int, optional
+        Mesh index
+    camera : int, optional
+        Camera index.
+    """
+    node = gltflib.Node()
+    if mesh is not None:
+        node.mesh = mesh
+    if camera is not None:
+        node.camera = camera
+    gltf.nodes.append(node)
+
+
+def write_mesh(gltf, primitives):
+    """Create mesh and add primitive.
+
+    Parameters
+    ----------
+    gltf : GLTF2
+        Pygltflib GLTF2 object.
+    primitives : list
+        List of Primitive object.
+    """
+    mesh = gltflib.Mesh()
+    for prim in primitives:
+        mesh.primitives.append(prim)
+
+    gltf.meshes.append(mesh)
+
+
+def write_camera(gltf, camera):
+    """Create and add camera.
+
+    Parameters
+    ----------
+    gltf : GLTF2
+        Pygltflib GLTF2 object.
+    camera : vtkCamera
+        scene camera.
+    """
+    orthographic = camera.GetParallelProjection()
+    cam = gltflib.Camera()
+    if orthographic:
+        cam.type = "orthographic"
+    else:
+        clip_range = camera.GetClippingRange()
+        angle = camera.GetViewAngle()
+        ratio = camera.GetExplicitAspectRatio()
+        aspect_ratio = ratio if ratio else 1.0
+        pers = gltflib.Perspective()
+        pers.aspectRatio = aspect_ratio
+        pers.znear, pers.zfar = clip_range
+        pers.yfov = angle * np.pi/180
+        cam.type = "perspective"
+        cam.perspective = pers
+    gltf.cameras.append(cam)
+
+
+def get_prim(verts, indices, cols, tcoords, normals, mat):
+    """Return a Primitive object.
+
+    Parameters
+    ----------
+    verts : int
+        Accessor index for the vertices data.
+    indices : int
+        Accessor index for the triangles data.
+    cols : int
+        Accessor index for the colors data.
+    tcoords : int
+        Accessor index for the texture coordinates data.
+    normals : int
+        Accessor index for the normals data.
+    mat : int
+        Materials index.
+
+    Returns
+    -------
+    prim : Primitive
+        pygltflib primitive object.
+    """
+    prim = gltflib.Primitive()
+    attr = gltflib.Attributes()
+    attr.POSITION = verts
+    attr.NORMAL = normals
+    attr.TEXCOORD_0 = tcoords
+    attr.COLOR_0 = cols
+    prim.attributes = attr
+    prim.indices = indices
+    if mat is not None:
+        prim.material = mat
+    return prim
+
+
+def write_material(gltf, bct: int, uri: str):
```

Comment:
Same as above

Reply:
writing the entire parameter name for these functions can go very long, also these parameters are explained in the docstring too.